app-text/mupdf/mupdf-1.19.0.ebuild

```diff
@@ -122,32 +122,32 @@ src_install() {
 		domenu platform/debian/${PN}.desktop
 		doicon platform/debian/${PN}.xpm
 	else
-		rm docs/man/${PN}.1 || die
+		rm docs/man/"${PN}".1 || die "Failed to rm in src_install()"
```

Comment:
Note that ${PN} can't contain spaces, so I usually wouldn't bother quoting it.

I'd say it arguably looks cleaner to do the whole string if you're so inclined, or just leave it.

Reply:
On the die message, maybe make it more specific ("couldn't cleanup man page") or similar? same applies to others